### ISSUE TITLE
[Question] Use of --delete-after with move command and exclusion?

### ISSUE THREAD

**Issue by a5m0** [ncw/rclone#481]
General question:

I am currently using move with an excluded folder (shown below) since move functions like a copy+delete when there is an excluded folder, can I use the --delete-after flag so that files are not deleted off the source until the complete copy of all files is finished?

`rclone --min-age 30d --checksum --bwlimit 4M --exclude excludefolder/** move /home/user/localfiles/ remoteacd:server/remotefolder`

**Comment by ncw** [comment]
move always deletes files after the copy.

If there are any IO errors during the copy then move won't delete the files.

I'll make this a bit clearer in the docs.

**Comment by makdisse** [comment]
When moving a folder, is it possible to delete every file right after the file is copied instead of deleting the folder only after all files have been copied?

**Comment by ncw** [comment]
@makdisse Not at the moment, no. Do you want to make a new issue?

**Comment by ncw** [comment]
I've re-worked the move docs and hopefully made them clearer - I'm going to close this now
